docs/controls/access_control/ac-05.md

```diff
@@ -0,0 +1,52 @@
+* System owner manages the account access. Additionally, System owner provisions a new user account on request.
+
+Data creation, control, and security functions are completed through the TDP application by a user who has been authenticated and is active in the system. These roles include OFA admin and STT users. 
+* STT/Data Preppers collect TANF data and transmit final data to OFA through the TANF application.
+* OFA admins have the ability to upload data on behalf of STTs and upload data files locally into the web application. 
+* These users cannot provision new accounts and do not have access to the account management functionality.
+
+**Part b.**  
+
+**Developers**  
+  - Use GitHub for project planning as well as the hosting site for all code repositories.  
+  - Ensures all testing, linting and vulnerability scan checks are passed before committing code to the repository or deploy to any environment.  
+  - Ensures a consistent development environment by using Docker to set up a code based container 
+  - Develops code in Dev environment.  
+  - Currently, only a dev environment is in place. A staging environment (which could be called a test environment) and a production environment will be set up in future releases.  The process of dev, stage (testing) and release to production is planned for future releases.  However, all code and documentation committed to HHS has to be approved by the Product Manager and the HHS Tech Lead.  
+
+**Users (OFA Admin and STTs)**
+  - approves new users, updates profile information, inactivates, and reactivates users
+  - reads data over assigned region
+  - able to upload and submit new TANF reports, replace and resubmit TANF reports  
+  - access (to be defined) allows the development of reports as required
+
+**Part c.**   
+Granting access to the TDRS system is performed by the OFA Admin through a request in the application.  Users request access to be able to create a profile and submit/view report in the application.
+
+Access to the TDRS system occurs in two different ways: through the Django admin interface and through the TDRS application interface.  
+
+***Access to the Django admin interface***  
+
+Access directly to the Django interface occurs through a combination of flags and assigned permissions based on responsibilities.  
```

Comment:
We should specify that the `superuser` and `staff` flags are required

Reply:
Updated - added superuser and staff flags requirement